KomaMRIBase/src/datatypes/Phantom.jl

```diff
@@ -181,7 +182,8 @@ Creates a two-dimensional brain Phantom struct.
 - https://brainweb.bic.mni.mcgill.ca/brainweb
 
 # Keywords
-- `axis`: (`::String`, `="axial"`, opts=[`"axial"`, `"coronal"`, `"sagittal"`]) orientation of the phantom
+- `axis`: (`::String`, `="axial"`, opts=[`"axial"`, `"coronal"`, `"sagittal"`]) orientation
+    of the phantom
```

Comment:
Why the extra line?

Reply:
I was simply trying to adhere to the 92-character line length limit.